source/code/scxsystemlib/common/GetLinuxOS.sh

```diff
@@ -124,7 +124,7 @@ GetLinuxInfo() {
 
         # Do we have the (newer) os-release standard file?
         # If so, that trumps everything else
-        if [ -e "${EtcPath}/os-release" ]; then
+        if [ -e "${EtcPath}/os-release" -a -z $ReleaseFile ]; then
```

Comment:
Does it mean now we'll never use /etc/os-release file to read the information?

Reply:
no, if `$ReleaseFile` is empty, it will use `/etc/os-release`.

Reply:
it seems we need to combine values from both `/etc/os-release` and `$ReleaseFile` if they both exist.

Reply:
In my think $ReleseFile will never be empty in practical scenarios. Please understand what would be the impact of this commit to https://github.com/microsoft/pal/commit/1d8d0016

Reply:
sure, I will update my PR later to combine values from 2 files if they both exist.